ruby_test.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class TestHound
+  NAME = "Wormbots"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.